platform/server/http/HttpServer.ts

```diff
@@ -27,11 +27,13 @@ export class HttpServer {
   }
 
   async start(config: HttpConfig) {
-    this.server = this.initializeServer(config);
+    const server = this.initializeServer(config);
```

Comment:
Not 100% sure why, but it complained in the `promisify` below that `this.server` could be `undefined`. It might be because TS can't guarantee what happens when it's used in `newPlatformProxyListener`.

Reply:
hmm, weird, it seems it doesn't complain for me, or maybe I'm doing something wrong :)